src/views/catalog/EnableInstanceTypeTechPreviewModal/EnableInstanceTypeTechPreviewModal.tsx

```diff
@@ -0,0 +1,149 @@
+      }
+      isOpen={isOpen}
+      onClose={onClose}
+      variant={ModalVariant.medium}
+    >
+      <Trans t={t} ns="plugin__kubevirt-plugin">
+        <Stack hasGutter>
+          <StackItem>
+            <Text component={TextVariants.p} className="pf-u-font-size-xl pf-u-danger-color-100">
+              Creating VirtualMachines is easier than ever!
+            </Text>
+          </StackItem>
+          <StackItem>
+            <Text component={TextVariants.p}>
+              This{' '}
+              <Text
+                component={TextVariants.a}
+                href="https://access.redhat.com/support/offerings/techpreview"
+                target="_blank"
+              >
+                Developer Preview feature
+              </Text>{' '}
+              provides a simple and quick way to create VirtualMachines.{' '}
+              <Text component={TextVariants.p}>
+                You must have cluster admin permissions to enable it. You can disable this feature
+                by going
+              </Text>{' '}
+              <Text component={TextVariants.p}>to the settings tab in the overview page.</Text>
+            </Text>
+          </StackItem>
+        </Stack>
+      </Trans>
+      <br />
+      <Split>
+        <SplitItem>
+          <Stack hasGutter>
+            <StackItem>
+              <Button
+                variant={ButtonVariant.link}
+                icon={<ExternalLinkAltIcon />}
+                //   href={}
+                target="_blank"
+                component="a"
+                iconPosition="right"
+                isInline
+              >
+                {t('Learn more about InstanceTypes')}
+              </Button>
+            </StackItem>
+            <StackItem>
+              <Button
+                variant={ButtonVariant.link}
+                component="a"
+                href="/quickstart?quickstart=creating-virtual-machine-from-volume"
+                isInline
+              >
+                {t('Quick start: Create a VirtualMachine from a volume')}
+              </Button>
+            </StackItem>
+            <StackItem isFilled />
+            <StackItem>
+              <Split hasGutter>
+                <Button
+                  variant={ButtonVariant.primary}
+                  isDisabled={!isAdmin}
+                  onClick={() => {
+                    toggleInstanceTypesFeature(true);
+                  }}
+                >
+                  {t('Enable')}
+                </Button>
+                <Button variant={ButtonVariant.link} onClick={onClose}>
+                  {t('Go to catalog')}
+                </Button>
+              </Split>
+            </StackItem>
+          </Stack>
+        </SplitItem>
+        <SplitItem isFilled />
+        <SplitItem>
+          <img
+            className="pf-u-pr-lg pf-u-pb-lg"
+            src={instanceTypeTechPreviewModalImage}
+            width={320}
```

Comment:
I'd suggest to explicitly provide also height of the image. It helps browsers to load the images faster. You could add a css class where you specify it, no need to add width as `img` element prop. But it's up to you.

Also I am not sure what exactly those patternfly classes mean, but those are not saying info about the dimensions of the image for sure. Maybe padding :) Maybe it could be all in some css file?

Reply:
> I'd suggest to explicitly provide also height of the image. It helps browsers to load the images faster. 
> You could add a css class where you specify it, no need to add width as `img` element prop. But it's up to you.

will add the height, just a quick note - you're right that it might help to load faster, but there's some some difference between adding the dimensions as props or as css attr - as props we get slightly better performence than in css file, but css file allows to handle dynamic sizes , I will add as prop for now.

> Also I am not sure what exactly those patternfly classes mean, but those are not saying info about the dimensions of the image for sure. Maybe padding :) Maybe it could be all in some css file?

both classes is for padding-right and padding bottom I know that these css classes [from PF](https://www.patternfly.org/v4/utilities/spacing/#usage) are not easy to read, but this is planned to be backported to 4.13.1 and I wanted to simplify the backport process by having less files to backport, hope that makes sense to you